tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerFilePath.fs

```diff
@@ -0,0 +1,65 @@
+namespace Test
+
+open System.Runtime.CompilerServices
+open CSharpLib
+
+type MyTy([<CallerFilePath>] ?p0 : string) =
+    let mutable p = p0
+
+    member x.Path with get() = p
+
+    static member GetCallerFilePath([<CallerFilePath>] ?path : string) =
+        path
+
+module Program =
+    let matchesPath path (s : string) =
+        s.EndsWith(path)
+          && not (s.Contains("\\\\"))
+          && not (path.Contains("\\.\\"))
+          && not (path.Contains("\\..\\"))
+
+        
+    [<EntryPoint>]
+    let main (_:string[]) =
+        let o = MyTy()
+        let o1 = MyTy("42")
+
+        match o.Path with
+        | Some(path) when matchesPath "Conformance\\SpecialAttributesAndTypes\\Imported\\CallerInfo\\CallerFilePath.fs" path -> ()
```

Comment:
the `\` is ok for windows, but on other os?
it's possibile to compare full path or normalize to uri? 
or use `__SOURCE_FILE__`